codestodians/food-basket-front-end
# Fix spontatneous-store-update bug.

#### What does this PR do?
- Fixes the recurrent `<Provider> does not support changing store on the fly` console warning.

#### Description of Task to be completed?
- Abstract the store and `apollo-client` into their individual directories.
- Remove the `Provider and store` from `App.jsx`, transferring them to the entry point of the application; `src/index.js`.
- Remove unused package in `src/index.js`

#### How should this be manually tested?
- Make changes (as you desire in any component), notice that you do not encounter the `<Provider> does not support changing store on the fly` warning anymore.

#### Any background context you want to provide?
- The `Provider` should not be passed into the `App` component that re-renders several times because of the hot-reloading feature. Rather, it should wrap the `App` component inside the `AppContainer` in the application's entry point. Only then will it (the provider) not be updated indiscriminately.

#### What are the relevant pivotal tracker stories?
[#153843656](https://www.pivotaltracker.com/story/show/153843656)
